icevision/models/mmdet/models/mask_rcnn/backbones.py

```diff
@@ -0,0 +1,75 @@
+__all__ = [
+    "r50_caffe_fpn_1x",
+    "r50_fpn_1x",
+    "r50_fpn_2x",
+    "r101_caffe_fpn_1x",
+    "r101_fpn_1x",
+    "r101_fpn_2x",
+    "x101_32x4d_fpn_1x",
+    "x101_32x4d_fpn_2x",
+    "x101_64x4d_fpn_1x",
+    "x101_64x4d_fpn_2x",
+]
+
+from icevision.models.mmdet.utils import *
+
+
+r50_caffe_fpn_1x = MMDetBackboneConfig(
+    model_name="mask_rcnn",
```

Comment:
`model_name` seems unnecessary, we only have it here because to do `save_dir = Path("checkpoints") / model_name` under `create_model_config`. But `weights_url` already have the model name included, i.e. `mask_rcnn_r50_caffe_fpn...`

Also, we can extract `model_name` from `cfg_url` if we really want to

Reply:
That's seems to be true for all MMDetection pretrained weights filenames. But if a user create its pretrained  pretrained weights filename without including the model name we cannot extract the name anymore.